### PR TITLE
feat(ios): add Amplify iOS Developer Preview

### PR DESCRIPTION
**Pull request by ameter**
_Description of changes:_
feat(ios): add Amplify iOS Developer Preview

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/fragments/lib/devpreview/ios/getting_started/10_intro.mdx [hunk 1]
```diff
@@ -0,0 +1,7 @@
+The Developer Preview of the Amplify iOS Library is now exclusively using Swift and provides  developers the ability to add cloud-based Auth, Storage, Data, and APIs to their apps. With this version, Swift developers will be able to debug and contribute to the underlying open-source codebase completely in Swift. We plan to incrementally add more Amplify features including Swift-based language features like structured concurrency.
```
Comment (siegerts) [inline src/fragments/lib/devpreview/ios/getting_started/10_intro.mdx:1]:
```suggestion
The Developer Preview of the Amplify iOS Library is now exclusively using Swift and provides developers the ability to add cloud-based Auth, Storage, Data, and APIs to their apps. With this version, Swift developers will be able to debug and contribute to the underlying open-source codebase completely in Swift. We plan to incrementally add more Amplify features including Swift-based language features like structured concurrency.
```

File: src/fragments/lib/auth/ios/escapehatch/10_awsmobileclient_escape.mdx [hunk 2]
```diff
@@ -53,4 +68,40 @@ awsmobileclient.federatedSignIn(providerName: IdentityProvider.apple.rawValue,
...
+```swift
+import AWSCognitoAuthPlugin
+import AWSCognitoIdentityProvider
+```
+
+Then retrieve the escape hatch with this code:
+
+```swift
+func getEscapeHatch() {
+    let client: CognitoIdentityProviderClient?
```
Comment (atierian) [inline src/fragments/lib/auth/ios/escapehatch/10_awsmobileclient_escape.mdx:88]: Optionality unnecessary as all logical branches are covered.
```suggestion
    let client: CognitoIdentityProviderClient
```

File: src/fragments/lib/devpreview/ios/getting_started/10_intro.mdx [hunk 3]
```diff
@@ -0,0 +1,7 @@
+The Developer Preview of the Amplify iOS Library is now exclusively using Swift and provides developers the ability to add cloud-based Auth, Storage, Data, and APIs to their apps. With this version, Swift developers will be able to debug and contribute to the underlying open-source codebase completely in Swift. We plan to incrementally add more Amplify features including Swift-based language features like structured concurrency.
```
Comment (atierian) [inline src/fragments/lib/devpreview/ios/getting_started/10_intro.mdx:1]: > ... ability to add cloud-based Auth, Storage, Data, and APIs to their apps.

Is it accurate that cloud-based Data and APIs can be added exclusively using Amplify iOS Developer Preview in combination with the Swift SDK?

File: src/fragments/lib/devpreview/ios/getting_started/30_preReq.mdx [hunk 4]
```diff
@@ -0,0 +1,8 @@
+* [Install and configure Amplify CLI](https://docs.amplify.aws/cli/start/install)
+* An iOS application targeting at least iOS 13.0
```
Comment (atierian) [inline src/fragments/lib/devpreview/ios/getting_started/30_preReq.mdx:2]: Should we add a minimum Xcode version (Swift version) here to be consistent with our existing pre-req section?

**Pull request merged**